packages/web-app-admin-settings/src/components/Users/UsersList.vue

```diff
@@ -4,10 +4,17 @@
       id="users-filter"
       v-model="filterTerm"
       class="oc-ml-m oc-my-s"
-      :label="$gettext('Filter users')"
+      :label="$gettext('Search')"
```

Comment:
Could we make it consistent, for groups and spaces as well?

Reply:
Good point!